--- a/lios/main.py
+++ b/lios/main.py
@@ -1327,23 +1327,15 @@ class linux_intelligent_ocr_solution():
 				self.dict = dictionary.Dict("en")
 				dlg = dialog.Dialog(_("Dictionary not found!"), (_("Ok"),dialog.Dialog.BUTTON_ID_1))
 				label = widget.Label(_(
- """Please install aspell, ispell, hunspell, myspell, or uspell 
+ """Please install aspell or myspell
 dictionary for your language and restart Lios!
 Otherwise spellchecker and auto-rotation will work with english(fallback). 
 
-For example on debian based system one can install aspell or 
-hunspell french dictionary using following commands
-apt-get install aspell-fr
-apt-get install hunspell-fr
-		
-or ispell dict using 
-apt-get install ifrench 
- 
-On rpm based system use 
-yum install aspell-fr
-			
-On arch based system use 
-pacman -S aspell-fr"""))
+For example on OpenMandriva based system one can install aspell or 
+myspell french dictionary using following commands
+sudo dnf install aspell-fr
+sudo dnf install myspell-fr
+"""))
 				dlg.add_widget(label)
 				label.show()
 				dlg.run()
